src/ImageSharp/Formats/Jpeg/GolangPort/OrigJpegDecoderCore.cs

```diff
@@ -505,8 +455,7 @@ private void ProcessApp1Marker(int remaining)
             byte[] profile = new byte[remaining];
```

Comment:
I wonder if we can reuse `this.Temp`  on these locations.

Reply:
Unfortunately not, it's far too large. I'd like to investigate pooling and copying the stream though, to an instance contained within the `ExifProfile` class.